### ISSUE TITLE
PHP's Cakephp - Validation Failed 

### ISSUE THREAD

**Issue by LadyMozzarella** [TechEmpower/FrameworkBenchmarks#1406]
Test 'cakephp' failing in preview 3 of round 10 with [json serialization, single query, and multiple queries](https://github.com/TechEmpower/TFB-Round-10/blob/master/peak/linux/results-2015-03-06-peak-preview3/latest/logs/cakephp/out.txt). All pass/warn locally through vagrant.

```bash
--------------------------------------------------------------------------------
  VERIFYING DB
--------------------------------------------------------------------------------
Accessing URL http://10.0.3.2:8080/world/index.json 
Response: 
"HTTP/1.1 500 Internal Server Error
Server: nginx/1.4.1
Date: Sat, 07 Mar 2015 01:41:49 GMT
Content-Type: text/html
Transfer-Encoding: chunked
Connection: keep-alive
X-Powered-By: PHP/5.5.17

"
   FAIL for http://10.0.3.2:8080/world/index.json
     Empty Response
--------------------------------------------------------------------------------
  VERIFYING JSON
--------------------------------------------------------------------------------
Accessing URL http://10.0.3.2:8080/json.json 
Response: 
"HTTP/1.1 500 Internal Server Error
Server: nginx/1.4.1
Date: Sat, 07 Mar 2015 01:41:49 GMT
Content-Type: text/html
Transfer-Encoding: chunked
Connection: keep-alive
X-Powered-By: PHP/5.5.17

"
   FAIL for http://10.0.3.2:8080/json.json
     Empty Response
```

**Comment by hamiltont** [comment]
Am I correct in seeing a pattern that tons of things are running inside vagrant but not running on the official test hardware? If so, what's the root cause?

**Comment by msmith-techempower** [comment]
If I had to guess, I'd say global system installs being broken in fantastic ways (example: Lapis from a few days ago)

**Comment by hamiltont** [comment]
Ugh, gross. So we prioritize #1387 for R11

**Comment by msmith-techempower** [comment]
Context: Lapis is a framework installed via an app called `luarocks` which is installed into the `IROOT`, but runs in tandem with Lua, which is installed via `apt-get`. To the best of my understanding, `luarocks` is a package/lib manager that is configured (hopefully, by default, but maybe overall) to install packages (like `lapis`) into `/usr/local/share/lua`.

When I Lapis was updated from 1.0.6-dev to 1.1.0 in the pull request, every environment that had a blank slate (read: travis, vagrant, etc) would simply be installing 1.1.0 for **the first time**, whereas our production environments are simply updated - the install scripts are meant to be atomic and able to be run again and again. Specifically, if `IROOT` is removed from a system and the entire suite is run again, the expectation is that 100% of the software required to run every test would be 100% installed again.

Presently, this is not true for a handful of languages/stacks (Haskell, Lua, etc).

**Comment by LadyMozzarella** [comment]
@hamiltont: JSYK - The only test that fails locally on vagrant for me is php-phalcon-mongodb (issue #1401). All of the other failing tests from preview 3 are passing.

**Comment by ssmith-techempower** [comment]
Resolved by #1461.
